labman/db/process.py

```diff
@@ -2378,17 +2380,18 @@ def generate_pool_file(self):
         str
             The contents of the pool file
         """
-        comp = self.components[0][0]
-        if isinstance(comp, composition_module.LibraryPrep16SComposition):
+        component_compositions = [x[0] for x in self.components]
```

Comment:
I think the proper way to test this is via of isinstance ....

Reply:
In this case, after the refactor, the code is actually comparing classes rather than comparing an instance to a class; in the former case, isinstance doesn't work so I followed the guidance at https://stackoverflow.com/a/23012248 re comparing classes directly.  I suppose one might argue that the approach of comparing classes is undesirable ...